_data/schedule.yml

```diff
@@ -0,0 +1,26 @@
+events:
+  -
+    title: Roundtable
+    location: CreatorSpace (typically)
+    address: 320 Railroad Ave.
+    topic:The first part of the meeting is educational with a speaker invited to
+      present on a niche topic that is of interest to entrepreneurs.
+
+      The second half of the meeting is an opportunity for an entrepreneur to
+      gain collective input from the group.  The entrepreneur has the
+      opportunity to present a new business idea, problem with their business,
+      or area where they are seeking help to a group of fellow entrepreneurs. The
+      group then has the chance to provide feedback and ask questions of
+      the presenter.
+    day: 1st Thursday of each month
+    time: 7:00 PM
+
+  -
+    title: Biz and Brews
+    location: Local Establishments
+    address: Changes Monthly (check Meetup)
```

Comment:
If this address field is used in the `schedule.html` file below, it may yield strange results in Google map searches.